pkgs/applications/misc/blender/default.nix

```diff
@@ -98,6 +112,9 @@ stdenv.mkDerivation rec {
 
   cmakeFlags =
     [
+      "-DWITH_GHOST_WAYLAND=ON"
+#      "-DWITH_GHOST_WAYLAND_LIBDECOR=ON"
+      "-DWITH_GHOST_WAYLAND_DYNLOAD=OFF"
```

Comment:
According to upstream comments and documentation, _DYNLOAD=OFF means it can't start on Xorg whilst also containing the ability to start on Wayland. If we ship with _DYNLOAD=OFF this apparently means you can only launch Blender on Wayland.

Reply:
Then I guess this should be fine? It can always be reverted.

Reply:
The comment on this commit https://projects.blender.org/blender/blender/commit/e58e023e1a implies that setting it to OFF would prevent falling back to X11, maybe I am misunderstanding it, or it is written in a confusing way?

 > When enabled, systems without Wayland installed will fall back to X11.

In this case, we are not enabling it, therefore does that mean we cannot fallback to X11?